File: app/src/main/java/com/tefsalkw/activity/SignUpThankYouActivity.java
package com.tefsalkw.activity;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import com.tefsalkw.R;

public class SignUpThankYouActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_signup_thank_you);


        Button buttonLogin = findViewById(R.id.buttonLogin);
        buttonLogin.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                Intent intent = new Intent(SignUpThankYouActivity.this, SigninActivity.class);
                startActivity(intent);
            }
        });


        String email = "";
        Intent intent = getIntent();
        if (intent != null) {
            email = intent.getStringExtra("email");

        }
        TextView txtStatus = findViewById(R.id.txtStatus);
        txtStatus.setText(getString(R.string.thank_you1) + email + getString(R.string.thank_you2));


    }
}

File: app/src/main/java/com/tefsalkw/kotlin/SignupThankYouActivity.kt
package com.tefsalkw.kotlin

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.tefsalkw.R
import com.tefsalkw.activity.SigninActivity
import kotlinx.android.synthetic.main.activity_signup_thank_you.*

class SignupThankYouActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_signup_thank_you)


        supportActionBar?.setDisplayShowTitleEnabled(false)

        val email =   intent.getStringExtra("email")

        txtStatus.text = "We have sent an activation link to your email address "+email+ ". Please activate your account to login."

        buttonLogin.setOnClickListener {



            val intent = Intent(this, SigninActivity::class.java)
                    //.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
                   // .addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK)
            startActivity(intent)


        }


    }
}
